source/jormungandr/jormungandr/scenarios/journey_filter.py

```diff
@@ -270,66 +270,74 @@ def get_nb_connections(journey):
     return journey.nb_transfers
 
 
-def way_later(request, journey, asap_journey):
-    """
-    to check if a journey is way later than the asap journey
-    we check for each journey the difference between the requested datetime and the arrival datetime
-    (the other way around for non clockwise)
+def _get_mode_weight(journey):
+    if 'car' in journey.tags:
+        return 4
+    if 'bike' in journey.tags:
+        return 3
+    if 'bss' in journey.tags:
+        return 2
+    return 1
```

Comment:
MODE_WEIGHT = {'car': 4, 'bike': 3, 'bss': 2}
return max([MODE_WEIGHT.get(mode, 1) for mode in journey.tags])